cura.desktop.in

```diff
@@ -10,6 +10,6 @@ TryExec=@CMAKE_INSTALL_FULL_BINDIR@/cura
 Icon=cura-icon
 Terminal=false
 Type=Application
-MimeType=application/sla;application/vnd.ms-3mfdocument;application/prs.wavefront-obj;image/bmp;image/gif;image/jpeg;image/png;
+MimeType=application/sla;application/vnd.ms-3mfdocument;application/prs.wavefront-obj;image/bmp;image/gif;image/jpeg;image/png;model/x3d+xml
```

Comment:
You miss a `;` at the end

Reply:
@petterreinholdtsen If you use `desktop-file-validate` to validate this file, it will complain about this... We had a PR #2295 which fixed this issue.